src/message_interface/message.rs

```diff
@@ -88,9 +88,9 @@ pub enum Message {
         /// The number of addresses to search for, after the last address with unspent outputs, in
         /// each account.
         address_gap_limit: u32,
+        #[serde(rename = "syncOptions")]
         /// Optional parameter to specify the sync options. The `address_start_index` and `force_syncing`
         /// fields will be overwritten to skip existing addresses.
-        #[serde(rename = "syncOptions")]
```

Comment:
I think @thibault-martinez will not like this :trollface:

Reply:
there is already a rename for this field at line https://github.com/iotaledger/wallet.rs/pull/1416/files/2036e55e5dbd78789fac96dbbd5e0957aa78cdae#diff-77379f45d1d6b511de579838dc1113ef09249aa75ca9450c9c5fac8164adae66R91

Reply:
it was a duplicate

Reply:
there is already a rename for this field a few lines above https://github.com/iotaledger/wallet.rs/pull/1416/files/2036e55e5dbd78789fac96dbbd5e0957aa78cdae#diff-77379f45d1d6b511de579838dc1113ef09249aa75ca9450c9c5fac8164adae66R91

Reply:
so i guess its fine to remove this one? :huh:

Reply:
You remove it and add it above the comment :huh:

Reply:
yeah because the other renames of the struct are above the comment too! :huh: